src/sentry/tasks/derive_code_mappings.py

```diff
@@ -94,11 +94,18 @@ def derive_code_mappings(
 
     if (
         not features.has("organizations:derive-code-mappings", org)
+        or not data.get("platform")
         or not data["platform"] in SUPPORTED_LANGUAGES
     ):
```

Comment:
purely opinion based, so take with grain of salt: pulling "not" outside makes it slightly easier to read. 
```suggestion
    if not (
        features.has("organizations:derive-code-mappings", org)
        and data.get("platform")
        and data["platform"] in SUPPORTED_LANGUAGES
    ):
```

Reply:
agreed, this is cleaner and more intuitive